### DISCUSSION TITLE
How to change selected item color in list without redesign the list

### DISCUSSION THREAD

**Discussion by Abod-Shashtari** [charmbracelet/bubbletea#437] [Q&A] [answered]
How can I change the selected item color in list from the default (purple) color to another color while keeping the default design as it was. e.g. : `m.list.Styles.Title=titleStyle;` here I have changed the title style how can I do something like this on list items.

**Comment by bashbunni** [comment, marked as answer]
Looks like it's defined [here](https://github.com/charmbracelet/bubbles/blob/b74f696cc8199d2d8d8d74e9484db6010baf76ca/list/defaultitem.go#L46) in the source code. 

So basically what you can do is tweak the styles defined in by `list.DefaultDelegate` to tailor the list items to suit your needs. Here's an example where I changed the selected item to be highlighted in red instead of the default pink: https://gist.github.com/bashbunni/0ab98bc9641223f611139382485b1ad3

Note that your items will need to satisfy the [`list.DefaultItem`](https://github.com/charmbracelet/bubbles/blob/b74f696cc8199d2d8d8d74e9484db6010baf76ca/list/defaultitem.go#L65) interface to display the `list.DefaultDelegate` styles correctly

**Reply by Abod-Shashtari** [reply]
Thank you :)
That was really helpful !
